applications/GeoMechanicsApplication/tests/test_benchmark_set_1.py

```diff
@@ -118,9 +118,6 @@ def test_benchmark1_3(self):
         reaction_force = [(cauchy_stress_at_load[0][3]+cauchy_stress_at_load[1][3]+cauchy_stress_at_load[2][3])/3*step_size for cauchy_stress_at_load in cauchy_stresses_at_load]
         cauchy_stress_tensor[min_distance_idxs]
 
-
-    # todo
-    @KratosUnittest.skip("test should be checked")
```

Comment:
Do we want to modify the Python test in this PR or will it be done another time? The parameters in this test can be better explained. Also, do we want to keep the test in this benchmark set?

Reply:
We started documenting this test due to a user question. Now that we have it running and the comparison with the Terzaghi solution is working, we would like to have it running nightly. therefore the activation. 

Moving it out of the non-descriptive benchmark set 1 is a good suggestion. Please show what can be better explained, the README.md file was created for that.

Reply:
Decided to do the split of and possible annotation/improvement of the python part later.